src/stages/models/helper.ts

```diff
@@ -68,7 +68,7 @@ function getCurrentPercentage(stageSummary: StageSummary): number {
  * @param stage The stage object containing the initial state flag.
  * @returns The initial xstate snapshot for the stage, Created as default or Wait if flag is true.
  */
-function getInitialXstate(stage: StageCreateWithTasksModel): Snapshot<unknown> {
+function getInitialXstate(stage: StageCreateModel): Snapshot<unknown> {
```

Comment:
maybe its better if theres no return type on this function? better generics and stuff

Reply:
done as we decide with dedicated type